examples/metadata_demo.ipynb

```diff
@@ -55,46 +55,51 @@
    "outputs": [],
    "source": [
     "# For local testing use\n",
-    "INGRESS = \"http://localhost:8080/seldon-deploy/api/v1alpha1\"\n",
+    "# INGRESS = \"http://localhost:8080\"\n",
     "\n",
     "# For production usecase set this to your ingress\n",
     "# for example if your SD URL is http://xxx.yyy.zzz.xyz/seldon-deploy/api/v1alpha1 set\n",
-    "# INGRESS = \"http://xxx.yyy.zzz.xyz\""
+    "INGRESS = \"http://34.79.179.26\""
```

Comment:
Maybe we can remove the specific IP address

Reply:
Sorry, That shoudl be xxx.yyy.zzz.xyz. Nice spot. Fixing

Reply:
Done